mirage/serializers/internal-user.ts

```diff
@@ -0,0 +1,37 @@
+import { ModelInstance } from 'ember-cli-mirage';
+import config from 'ember-osf-web/config/environment';
+
+import InternalUserModel from 'ember-osf-web/models/internal-user';
+
+import ApplicationSerializer from './application';
+
+const { addonServiceUrl } = config.OSF;
+
+export default class InternalUserSerializer extends ApplicationSerializer<InternalUserModel> {
+    buildRelationships(model: ModelInstance<InternalUserModel>) {
+        return {
+            authorizedStorageAccounts: {
+                links: {
+                    related: {
+                        href: `${addonServiceUrl}v1/internal_users/${model.id}/authorized_storage_accounts`,
```

Comment:
I wonder if we should use `app/adapters/addon-service.ts`'s namespace in all these. I'm mostly just worried that they may decide to go with v0 then we have to change it in a lot of places. Not that you wouldn't have to change it now to be prepared for it, so it wouldn't really save you any trouble unless they change it twice. Which they coooould do.